### ISSUE TITLE
Use color from the current theme

### ISSUE THREAD

**Issue by dannycolin** [cmcaine/tridactyl#391]
I use [VIvaldiFox][vf] to change the color of firefox via the Theme API but I've found that the dark theme of Tridactyl seems to be hardcoded because it doesn't apply the change I've made in VivaldiFox.

Would it be possible to load the current color with the Theme API and set the color of the console and find mode with it?

[vf]:https://github.com/nt1m/vivaldi-fox

**Comment by bovine3dom** [comment]
Yes, it is possible: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/theme/getCurrent

It will be quite far down our list of priorities, though. 

If anyone who wants this wants to have a go implementing it, we'd be happy to help them on Matrix (details on readme.md).

**Comment by dannycolin** [comment]
Thanks for the quick response.

I know nothing about webext but I have some knowledge in programmation. I'll try to take a look at it as soon as my final exams are finish. :)

**Comment by caputchinefrobles** [comment]
Has anyone volunteered for this task already? I'd like to volunteer if @dannycolin is not already up to this.

**Comment by dannycolin** [comment]
@caputchinefrobles go ahead. I'm too busy at the moment and there'll be other issues to work on ;)

**Comment by caputchinefrobles** [comment]
Starting with this issue today

**Comment by surenzxx** [comment]
any update on this?

**Comment by bovine3dom** [comment]
@surenzxx you can see activity on forks here https://github.com/tridactyl/tridactyl/network so it looks like no-one is working on this.

We'd be happy to help you get started if you're interested!

**Comment by basbebe** [comment]
Any progress here?

I would like so set my tridactyl theme depending on the system theme (switch between light / dark mode)…
